interface-definitions/include/qos/mtu.xml.i

```diff
@@ -0,0 +1,15 @@
+<!-- include start from qos/mtu.xml.i -->
+<leafNode name="mtu">
+  <properties>
+    <help>MTU size for this class</help>
+    <valueHelp>
+      <format>&lt;number&gt;</format>
+      <description>Bytes</description>
+    </valueHelp>
+    <valueHelp>
+      <format>&lt;number&gt;&lt;suffix&gt;</format>
+      <description>Bytes with scaling suffix (kb, mb, gb)</description>
```

Comment:
Does it actually accept gigabyte-large MTUs? Even if yes — should we allow that?

Reply:
> Does it actually accept gigabyte-large MTUs? Even if yes — should we allow that?

I'm unaware of a Gigabyte MTU 

Please use the common MTU definition instead: https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/interface/mtu-68-16000.xml.i

Reply:
68 is too low for tc possible value is 256 - 4294967295
```
vyos@vyos:~$  tc filter replace dev eth0 parent ffff: prio 255 protocol all basic action police conform-exceed drop/ok rate 10000000 burst 100kb mtu 68
RTNETLINK answers: Cannot allocate memory
We have an error talking to the kernel
vyos@vyos:~$  tc filter replace dev eth0 parent ffff: prio 255 protocol all basic action police conform-exceed drop/ok rate 10000000 burst 100kb mtu 255
RTNETLINK answers: Cannot allocate memory
We have an error talking to the kernel
vyos@vyos:~$  tc filter replace dev eth0 parent ffff: prio 255 protocol all basic action police conform-exceed drop/ok rate 10000000 burst 100kb mtu 256
vyos@vyos:~$  tc filter replace dev eth0 parent ffff: prio 255 protocol all basic action police conform-exceed drop/ok rate 10000000 burst 100kb mtu 4294967295
vyos@vyos:~$  tc filter replace dev eth0 parent ffff: prio 255 protocol all basic action police conform-exceed drop/ok rate 10000000 burst 100kb mtu 4294967296
Error: argument "4294967296" is wrong: mtu
```

in the task description, author said `the packet size between ip stack and nic could be 65536` therefore range 68-16000 probably is incorrect for this case, I think we can use range 256-65535.

Reply:
Then please limit to this range

Reply:
done